open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java

```diff
@@ -0,0 +1,246 @@
+        HBox r1 = thecontrols.createCustomText("Shift");
+        r1.getChildren().addAll(new ImageView(myPlusDir), thecontrols.customImageView(mouseLeft));
+
+        VBox l2 = thecontrols.createLeftLabel("Toggle Arc Length");
+        HBox r2 = thecontrols.createCustomText("m");
+
+        VBox l3 = thecontrols.createLeftLabel("Live Track Mode");
+        HBox r3 = thecontrols.createCustomText("Ctrl [hold]");
+
+        VBox l4 = thecontrols.createLeftLabel("Bullseye Vector");
+        HBox r4 = thecontrols.createCustomText(",");
+
+        VBox l5 = thecontrols.createLeftLabel("Copy Coordinates");
+        HBox r5 = thecontrols.createCustomText("C");
+
+        VBox l6 = thecontrols.createLeftLabel("Display All Coordinates");
+        HBox r6 = thecontrols.createCustomText(".");
+
+        dispArea.add(l1, 0, 1);
+        dispArea.add(r1, 2, 1);
+        dispArea.add(l2, 0, 2);
+        dispArea.add(r2, 2, 2);
+        dispArea.add(l3, 0, 3);
+        dispArea.add(r3, 2, 3);
+        dispArea.add(l4, 0, 4);
+        dispArea.add(r4, 2, 4);
+        dispArea.add(l5, 0, 5);
+        dispArea.add(r5, 2, 5);
+        dispArea.add(l6, 0, 6);
+        dispArea.add(r6, 2, 6);
+        return mainBox;
+    }
+
+    /**
+     * Creates a sub-window containing the General Controls.
+     * @return mainBox the sub-window contained in a GridPane.
+     */
+    private GridPane createGeneral()
+    {
+        ControlTab controlT = new ControlTab();
+        GridPane mainBox = controlT.createBlankPane(" General Controls");
+        GridPane dispArea = controlT.getDisplayArea();
+
+        VBox l1 = controlT.createLeftLabel("Save State");
+        HBox r1 = controlT.createCustomText("Ctrl");
+        r1.getChildren().addAll(new ImageView(myPlusDir), controlT.createCustomText("S"));
+
+        VBox l2 = controlT.createLeftLabel("Undo");
+        HBox r2 = controlT.createCustomText("Ctrl");
+        r2.getChildren().addAll(new ImageView(myPlusDir), controlT.createCustomText("Z"));
+
+        VBox l3 = controlT.createLeftLabel("Redo");
+        HBox r3 = controlT.createCustomText("Ctrl");
+        r3.getChildren().addAll(new ImageView(myPlusDir), controlT.createCustomText("Y"));
+
+        VBox l4 = controlT.createLeftLabel("Collect Garbage ");
+        HBox r4 = controlT.createCustomText("g");
+
+        VBox l5 = controlT.createLeftLabel("Cancel Query");
+        HBox r5 = controlT.createCustomText("Esc");
+
+        dispArea.add(l1, 0, 1);
+        dispArea.add(r1, 2, 1);
+        dispArea.add(l2, 0, 2);
+        dispArea.add(r2, 2, 2);
+        dispArea.add(l3, 0, 3);
+        dispArea.add(r3, 2, 3);
+        dispArea.add(l4, 0, 4);
+        dispArea.add(r4, 2, 4);
+        dispArea.add(l5, 0, 5);
+        dispArea.add(r5, 2, 5);
+        return mainBox;
+    }
+
+    /**
+     * Creates a sub-window containing Map Zoom Controls.
+     * 
+     * @return mainBox the sub-window contained in a GridPane.
```

Comment:
No need to give the name of the return variable

Reply:
updated